src/modules/75-ce/pages/recommendationList/RecommendationList.tsx

```diff
@@ -87,6 +83,14 @@ const RecommendationsList: React.FC<RecommendationListProps> = ({
     }
   }, [])
 
+  const resourceTypeMap: Record<string, string> = useMemo(
+    () => ({
+      [ResourceType.Workload]: getString('ce.overview.workload'),
+      [ResourceType.NodePool]: getString('ce.overview.nodepool')
+    }),
+    []
```

Comment:
Codacy has a fix for the issue: [React Hook useMemo has a missing dependency: 'getString'. Either include it or remove the dependency array.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232309)
```suggestion
    [getString]
```